.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
Is this file required in addition to the `.dockerignore` in the `.binder` directory?

Reply:
Sorry for the delay! But yes, this is important to ignore similar artifacts for this directory. `.binder/.dockerignore` functions the same as `.gitignore` - it only applies to enclosed directories.